lib/storage/durable.js

```diff
@@ -1,10 +1,28 @@
-const PouchDB = require('pouchdb');
-const db = new PouchDB(__dirname + '/../../storage/pouch.db');
-
-module.exports = {
-  put: db.put,
-  get: db.get,
-  query: db.query,
-  remove: db.remove,
-  raw: db
-};
+const loki = require('lokijs');
+const fs = require('fs');
+
+const dbFile = __dirname + '/../../storage/iris.json'
+
+const db = new loki();
+
+module.exports = function(collectionName) {
+  let coll = db.getCollection(collectionName);
+  if(coll) return coll;
+  return db.addCollection(collectionName);
+}
+
+try {
+  let oldDB = fs.readFileSync(dbFile, {encoding: 'utf8'});
+  db.loadJSON(oldDB);
+} catch(e){
+  console.log('Failed to load old DB', e);
+}
+
+setInterval(()=>{
```

Comment:
1. We only write when the db is dirty
2. The overall file size is small (a few kb at worst)
3. It's over the span of a second which is pretty big
4. It's async, so if one write is held up, the next ones will just fail because there's a lock